boards/xtensa/intel_s1000_crb/Kconfig.defconfig

```diff
@@ -69,6 +69,37 @@ config SPI_0_IRQ_PRI
 config SPI_DW_FIFO_DEPTH
 	default 32
 
+config SPI_NOR
+	def_bool y
+
+config SPI_NOR_SPI_SLAVE
+	default 0
+
+config SPI_NOR_DRV_NAME
+	default "MX25"
```

Comment:
that name could come from DTS already, as well as the SPI frequency below. (use fixup to relate these.)